packages/client/package.json

```diff
@@ -93,6 +93,7 @@
     "@prisma/internals": "workspace:*",
     "@prisma/migrate": "workspace:*",
     "@prisma/mini-proxy": "0.9.4",
+    "@prisma/query-engine-wasm": "5.5.0-3.6e26301fe272ba4ba0598fe43eb5d8df030be4db",
```

Comment:
Fictitious package for now.

Reply:
It actually exists now 😆 
https://prisma-company.slack.com/archives/C05Q1T8RJ8K/p1696608960033699?thread_ts=1696568642.779789&cid=C05Q1T8RJ8K